src/containers/paper-canvas.jsx

```diff
@@ -114,13 +124,31 @@ class PaperCanvas extends React.Component {
             this.props.canvasRef(canvas);
         }
     }
+    handleWheel (event) {
+        if (event.metaKey) {
+            // Zoom keeping mouse location fixed
```

Comment:
It might be more useful to have this be the default behavior rather than the meta key behavior, since I don't think most people know you can convert your mouse scroll to a horizontal scroll, which means it'll be hard to move left/right otherwise. Of course we'd have to test it.

Reply:
Interesting idea. I think we'll have to see what it feels like with people. I think the trackpad behavior is good as the default, and I do not currently know a way to tell the difference between a real "mouse wheel" and a trackpad. If there was, I would totally swap them.